packages/@okta/vuepress-site/docs/guides/authenticators-okta-verify/main/java/enrollmentotherintegrationsteps.md

```diff
@@ -0,0 +1,103 @@
+#### 1. Initiate sign-in flow and return a list of authenticators
```

Comment:
Sequence diagram missing before this section.

Reply:
I missed this one, I updated figma and added a TODO inline